src/components/AppSettingsMenu.vue

```diff
@@ -4,6 +4,28 @@
 			:name="t('mail', 'Mail settings')"
 			:show-navigation="true"
 			:open.sync="showSettings">
+			<NcAppSettingsSection id="mail-list-view" :name="t('mail', 'Layout')">
+				<NcCheckboxRadioSwitch value="no-split"
+					name="mail-layout"
+					type="radio"
+					:checked="layoutMode"
```

Comment:
```suggestion
					:checked="layoutMode === 'no-split"
```

to revive the checked state?